src/pages/kb/user-guide/dashboards/dashboard-editing.md

```diff
@@ -77,4 +79,4 @@ Automatic Dashboard Refresh occurs as part of the Redash frontend application. Y
 
 {% endcallout %}
 
-Importantly, when visitors access a public dashboard they will always see the cached query result. Refreshing a public dashboard does **not** execute the underlying queries. The only way that visitors to a public dashboard link will see refreshed data is if a logged-in Redash user executes the underlying queries or if a query schedule is in effect.
+Importantly for users accessing a public dashboard, the Refresh button will **not** execute the underlying queries. It just checks the cache and updates the dashboard if it finds newer results. The only way that visitors to a public dashboard will see refreshed data is if a logged-in Redash user executes the underlying queries or if a query schedule is in effect.
```

Comment:
@rauchy is this still correct?

Reply:
Nope, public dashboards now execute queries when new parameter values are applied (assuming they are safe).

Reply:
I will update this then. I've restored the branch.